src/components/prescriptions/PrescriptionButtonComponent.vue

```diff
@@ -0,0 +1,284 @@
+<template>
+    <div
+        class="prescriptionBox"
+        @submit.prevent="addTest">
+        <h3 class="prescriptionBox__header">Dodaj receptę:</h3>
+        <form class="prescriptionBox__form prescriptionForm">
+            <div class="selectBox">
+                <label for="addManually">Dodaj ręcznie</label>
+                <input
+                    id="addManually"
+                    v-model="addManually"
+                    type="checkbox" />
+            </div>
+            <div class="selectBox">
+                <label for="import">Zaimportuj receptę</label>
+                <input
+                    id="import"
+                    v-model="importFlag"
+                    type="checkbox" />
+            </div>
+
+            <div v-show="addManually">
+                <div class="inputBox">
+                    <label for="nameAndSurnameDoctor"
+                        >Imię i nazwisko lekarza:</label
+                    >
+                    <select v-model="nameAndSurnameDoctor">
+                        <option
+                            disabled
+                            value="">
+                            Wybierz lekarza
+                        </option>
+                        <option
+                            v-for="research in getDoctorsArray()"
+                            :key="research.registryId">
+                            {{ research.doctorName }}
+                        </option>
+                    </select>
+                </div>
+                <div class="inputBox">
+                    <label for="medicine">Lek:</label>
```

Comment:
if u add for attr for label markup, you should add id attr for element which is indicated with this label